core/storage/base_model/gae_models.py

```diff
@@ -312,7 +312,7 @@ def get_all(cls, include_deleted=False):
         return query
 
     @classmethod
-    def get_new_id(cls, entity_name):
+    def get_new_id(cls, entity_name=''):
```

Comment:
In general please try to avoid using default args. They increase (implicit) branching and make the code harder to maintain. Please also try to avoid making changes to core infrastructure code if not necessary.